packages/bpk-scrim-utils/src/withScrimmedPortal-test.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render, within } from '@testing-library/react';
+
+import withScrimmedPortal from './withScrimmedPortal';
+
+describe('withScrimmedPortal', () => {
```

Comment:
Should we also test the thrown error scenario in these tests of it both the document.body or the provided target?

Reply:
Tried to add a test, but I think jest just won't render anything at all if document/body are not defined